docs/content/users/topics/sharing.md

```diff
@@ -20,8 +20,8 @@ There are CMSes that make this a little harder, especially WordPress and Magento
 
 ### Setting up a Stable ngrok Subdomain
 
-1. Get a paid token with at least the basic plan, and configure it. It will be in `~/.ngrok2/ngrok.yml` as `authtoken`.
-2. Configure `ngrok_args` to use a stable subdomain. In `.ddev/config.yaml`, `ngrok_args: --subdomain wp23` will result in ngrok always using `wp23.ngrok.io` as the URL, so it’s not changing on you all the time.
+1. Get a token with at least the free plan from [ngrok.com](https://ngrok.com/), and configure it. It will be in `~/.ngrok2/ngrok.yml` as `authtoken`.
+2. Configure `ngrok_args` to use a stable subdomain. In `.ddev/config.yaml`, `ngrok_args: --subdomain wp23` will result in ngrok always using `wp23.ngrok.io` as the URL, so it’s not changing on you all the time. This feature only works for paid ngrok plans.
```

Comment:
Why did you change the first item to say "free plan" and then in second item say that it only worked for paid plans?

Reply:
@rfay 
You can use ngrok with ddev on the free plan (point 1 in the list)
You can use the "stable subdomain" feature only on a paid plan.

Reply:
But this section is about stable subdomain...

Reply:
you're right. just decline the PR then?